projects/longest-word-in-string/_posts/2020-10-26-javascript.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Longest word in a string in JavaScript
```

Comment:
```suggestion
title: Longest Word in JavaScript
```

I think this name is probably sufficient (string is implied).  The file path will have to change as well.